File: 01 Introdução/1010 Cálculo Simples/1010.py
piece1 = input().split(" ")
piece2 = input().split(" ")
codigo1, quantidade1, valor1 = piece1
codigo2, quantidade2, valor2 = piece2

valorTotal = int(quantidade1) * float(valor1) + \
    int(quantidade2) * float(valor2)
print(f"VALOR A PAGAR: R$ {valorTotal:.2f}")

# https://www.beecrowd.com.br/judge/pt/problems/view/1010
# Ranking 25907º, Time 0.061, Submiss Date 03/02/2023 22:57:12 (Python 3.9)

File: 01 Introdução/1010 Cálculo Simples/1010.java
import java.util.Scanner;

class Main {

  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    sc.next();
    int quantityPiece1 = sc.nextInt();
    double valuePiece1 = sc.nextDouble();

    sc.next();
    int quantityPiece2 = sc.nextInt();
    double valuePiece2 = sc.nextDouble();

    System.out.printf("VALOR A PAGAR: R$ %.2f\n",
        quantityPiece1 * valuePiece1 + quantityPiece2 * valuePiece2);
    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1010
// Ranking 22489º, Time 0.625, Submiss Date 12/02/2023 22:33:31 (Java 14)